.github/labeler.yml

```
@@ -0,0 +1,5 @@
+assignments: assignments/**
+github: .github/**
+
+text: ['*.txt']
+python: ['*.py']
```

please add a newline here
```suggestion
python: ['*.py']

```